.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a bug report to help improve depstat
+title: ''
+labels: 'kind/bug'
+assignees: ''
+
+---
+
+<!-- Please answer these questions before submitting your bug report. Thanks! -->
+
+**What version of depstat are you using?**
+
+depstat version:
+
+**Does this issue reproduce with the latest release?**
```

Comment:
Is there a particular reason we are asking this explicitly?

If the author of the bug report would have tested against the latest depstat release, that would be captured in the first question, correct? :thinking:

Reply:
Aggreed. I based the template on other in the Kubernetes project world, and will remove line 16.